go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -54,9 +54,11 @@ func NewController(client clientset.Interface,
 	broadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: client.CoreV1().Events("")})
 	recorder := broadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: controllerName})
 
+	st := newServiceTracker()
```

Comment:
when the startup happens we get all of the events and end up idempotently adding back with may-exist anything that was missing. Does it not work the same way for your new controller?

Reply:
yep, I think that is how the informer works, it gives you a List and then a Watch

Reply:
However, I´m using the same ovn loadbalancer functions, so I assume that those are idempotent , maybe you are asking about this?